app/src/pages/save-the-date.tsx

```diff
@@ -0,0 +1,341 @@
+                                  style={{
+                                    height: "2rem",
+                                    margin: "0 0.5rem 1rem 0.5rem",
+                                  }}
+                                />
+                              </a>
+                            ))}
+                        </center>
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </center>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      end-tag
+    </div>
+  );
+}
+
+function TranslatedSection(
+  save_the_date: SaveTheDate,
+  cells: SaveTheDateCell[]
+) {
+  const language_code = locale(useRouter());
+  const commissions_cells = cells.filter((cell) => cell.commission);
+  const clic_cells = cells.filter((cell) => cell.commission == null);
+  const translated_std = getTranslation(save_the_date, language_code);
+  const tt = useTranslationTable();
+
+  return (
+    <>
+      <tr>
+        <td>
+          <center>
+            {save_the_date.image ? (
+              <img
+                src={directusImageUrl(save_the_date.image)}
+                alt="CLIC"
+                style={{ width: "100%", height: "200px" }}
+              />
+            ) : (
+              <></>
+            )}
+            <h1
+              style={{
+                color: save_the_date.title_color || undefined,
+                ...TITLE_STYLE,
+              }}
+            >
+              {tt["save-the-date"].toUpperCase()}
+            </h1>
+          </center>
+        </td>
+      </tr>
+
+      <tr>
+        <td style={{ padding: "1rem 0 1rem 0" }}>
+          <center>
+            <a
+              href="https://clic.epfl.ch/en-US/save-the-date/"
+              style={{
+                background: save_the_date.button_color || undefined,
+                ...BUTTON_STYLE,
+              }}
+            >
+              {tt["translated-version"]}
+            </a>
+          </center>
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          <center>
+            <table style={CELL_GROUP_STYLE}>
+              <tbody>{clic_cells.map((cell) => StdCellComponent(cell))}</tbody>
+            </table>
+          </center>
+        </td>
+      </tr>
+
+      <tr>
+        <td>
+          <center>
+            <h1
+              style={{
+                color: save_the_date.title_color || undefined,
+                ...TITLE_STYLE,
+              }}
+            >
+              {tt["commissions"].toUpperCase()}
+            </h1>
+          </center>
+        </td>
+      </tr>
+
+      <tr>
+        <td style={{ padding: 10 }}>
+          <table
+            style={{
+              border: "solid 1px " + save_the_date.title_color,
+              borderRadius: 25,
+              ...CELL_GROUP_STYLE,
+            }}
+          >
+            <tbody>
+              {commissions_cells.map((cell) => StdCellComponent(cell))}
```

Comment:
Maybe putting the name of the commission / grouping events per commission like in previous save the dates ? Here we don't know what event is what commission :)

I like the sort of frame you used, maybe we could have a frame like this per commission instead and just add their name + logo at the top, or something like that

Reply:
Yes it is a good idea, I'll try something :)